app/views/docs/production.phtml

```diff
@@ -13,38 +13,48 @@
+    <li>Only the root user can signup. All other developers must be added through invitations. This is configured using the <code>_APP_CONSOLE_WHITELIST_ROOT</code> environment variable.</li>
+</ol>
+
+<p>By default, only the first user can sign up on the Appwrite instance's dashboard. All other users must be added to the dashboard through invitation.</p>
+
+<p>
+<a href="/docs/environment-variables"><i class="icon-angle-circled-right"></i>Learn more about environment variables</a>
+</p>
 
 <h2><a href="/docs/production#scaling" id="scaling">Scaling</a></h2>
 
-<p>Appwrite was built with scalability in mind. Appwrite can scale both horizontally and vertically.</p>
+<p>Appwrite is built with scalability in mind. Appwrite can scale both horizontally and vertically.</p>
 
-<p>Appwrite uses a few containers to run, where each container has its job. Most of the Appwrite containers are stateless, and in order to scale them, all you need is to replicate them and setup a load balancer to distribute their load.</p>
+<p>Each Appwrite instance is composed of many containers, each with its unique job. Most of the Appwrite containers are stateless, and to scale them, all you need is to replicate them and set up a load balancer to distribute their load.</p>
 
 <p>If you decide to set up a load balancer for a specific container, make sure that the containers that are trying to communicate with it are accessing it through a load balancer and not directly. All connections between Appwrite different containers are set using Docker environment variables.</p>
 
-<p>There are three Appwrite containers that do keep their state. MariaDB, Redis, and InfluxDB containers are used for storing data, cache and pub/sub messaging, and usage stats (in this order). To scale them out, all you need to do is set up a standard cluster (same as you would with any other app using these technologies) according to your needs and performance.</p>
+<p> Three Appwrite containers are stateful. The MariaDB, Redis, and InfluxDB containers are used for storing data, cache and pub/sub messaging, and usage stats, respectively. To scale these containers, set up a standard cluster (same as you would with any other app using these technologies) according to your needs and performance.</p>
 
 <h2><a href="/docs/production#emails" id="emails">Emails</a></h2>
 
-<p>Sending emails is hard. There are a lot of SPAM rules and configurations to master in order to set a functional SMTP server. The SMTP server that comes packaged with Appwrite is great for development but needs some work done to function well against SPAM filters.</p>
+<p>Sending emails is hard. There are a lot of spam rules and configurations to master to set up a functional SMTP server. The SMTP server that comes pre-packaged with Appwrite is satisfactory for testing, but you'll need a third-party solution to deliver emails that do not get labeled as spam.</p>
```

Comment:
I'm actually not sure if Appwrite comes with an SMTP server 👀

Reply:
Oooooooo Wow we don't anymore! big bait!

Reply:
We have it on GItHub. But we don't release that. If you install Appwrite using official command, no smtp server. Not even if comment if I remember right.